feature/qos/otg_tests/ingress_police_nhg/README.md

```diff
@@ -0,0 +1,216 @@
+
+  input-policies:       # new OC subtree input-policies (/qos/input-policies)
+    - input-policy: "limit_group_A_2Gb"
+      config:
+        name: "limit_group_A_2Gb"
+        classifer: "dest_A"
+        scheduler-policy: "limit_2Gb"
+    - input-policy: "limit_dest_group_B_1Gb"
+      config:
+        name: "limit_dest_group_B_1Gb"
+        classifer: "dest_B"
+        scheduler-policy: "limit_1Gb"
+
+  interfaces:                  # this is repeated per subinterface (vlan)
+    - interface: "PortChannel1"
+      interface-ref:
+        config:
+          subinterface: 100
+    input:
+      config:
+        policies:  [            # new OC leaf-list (/qos/interfaces/interface/input/config/policies)
+          limit_dest_group_A_2Gb
+        ]
+  interfaces:                  # this is repeated per subinterface (vlan)
+    - interface: "PortChannel1"
+      interface-ref:
+        config:
+          subinterface: 200
+    input:
+      config:
+        policies:  [            # new OC leaf-list (/qos/interfaces/interface/input/config/policies)
+          limit_dest_group_B_1Gb
+        ]
+
+```
+
+### TE-18.3.1 Test traffic
+
+* Send traffic
+  * Send traffic from ATE port 1 to DUT for dest_A and is conforming to cir.
+  * Send traffic from ATE port 1 to DUT for to dest_B and is conforming to
+    cir.
+  * Validate packets are received by ATE port 2.
+    * Validate qos interface scheduler counters
+    * Validate afts next hop counters
+  * Validate outer packet ipv6 flow label assignment
+    * When the outer packet is IPv6, the flow-label should be inspected on the ATE.
+      * If the inner packet is IPv4, the outer IPv6 flow label should be computed based on the IPv4 5 tuple src,dst address and ports, plus protocol
+      * If the inner packet is IPv6, the inner flow label should be copied to the outer packet.
+  * Increase traffic on flow to dest_B to 2Gbps
+    * Validate that flow dest_B experiences ~50% packet loss (+/- 1%)
+
+# TE-18.3 Control maximum rate of traffic transmitted or received on an interface.
+
+## Summary
+
+Bandwidth Management Through Rate Limiting
+Class-based policing allows you to control the maximum rate of traffic transmitted or received on an interface. 
+Class-based policing is often configured on interfaces at the edge of a network to limit traffic into or out of the network.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+Use TE-18.1 test environment setup.
+
+## Procedure
+
+### TE-18.4.1 Generate and push configuration
+
+* Use default class (commonly known as the class-default class).
+* Configures a policer classifying on an input aggregate (lag) interface VLAN.
+* Add a classifier by input mac of DUT port 1 in order to match all inbound L2 traffic.
+* Add a limit after which traffic is dropped.
```

Comment:
Consider removing the references to 'class' and 'class-default' which are undefined concepts in OC QoS.

In OC, there are classifiers and schedulers.  Classifers are the match rule,  Schedulers can be configured to police traffic and also optionally shape traffic and optionally perform different de-queueing algorithms.  In this case, we are defining a scheduler with only a policer.